packages/mattermost-redux/src/selectors/entities/preferences.ts

```diff
@@ -18,7 +17,7 @@ import {getPreferenceKey} from 'mattermost-redux/utils/preference_utils';
 import {setThemeDefaults} from 'mattermost-redux/utils/theme_utils';
 
 export function getMyPreferences(state: GlobalState): { [x: string]: PreferenceType } {
-    return state.entities.preferences.myPreferences;
+    return state.entities.preferences?.myPreferences ?? {};
```

Comment:
Same question here. Also, we shouldn't be returning object literals from non-memoized selectors